GOVERNANCE.md

```diff
@@ -0,0 +1,64 @@
+<!--
+SPDX-FileCopyrightText: 2024 German Aerospace Center (DLR)
+SPDX-FileContributor: Stephan Druskat
+
+SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+## Governance
+
+The HERMES project employs a lightweight governance model.
+
+All decisions are made by the [HERMES Steering Group](#hermes-steering-group) 
+together with the maintainer(s) of the repositories affected by the decision.
+
+Decisions that adhere to a single [repository in the scope of the HERMES project](#hermes-project-repositories)
+are made by *lazy consensus*: 
+
+1. The maintainer of the repository makes a decision and notifies the HERMES steering group
+(using `@softwarepub/hermes-steering-group` mention).
+2. If no members of the steering group veto the decision within the decision period of one week (168 hours),
+the decision becomes effective and can be acted upon. 
+The maintainer and any member of the steering group can ask for a longer decision period, 
+which must be granted. 
+3. If one or more members of the steering group veto the decision, 
+the discussion is discussed until consensus is reached.
+
+Decisions that adhere to more than one [repository in the scope of the HERMES project](#hermes-project-repositories)
+must be based on full consensus within the steering group. 
+
+
+### HERMES project repositories
+
+The following are the repositories that are governed by the steering group. 
+
+- <https://github.com/softwarepub/hermes> (Maintainer: Stephan Druskat)
+- <https://github.com/softwarepub/ci-templates> (Maintainer: Oliver Bertuch)
+- <https://github.com/softwarepub/hermes-harvest-template> (Maintainer: tbc)

```

Comment:
tbd?

Reply:
@zyzzyxdonta Yes, no, yo: private repo, shouldn't appear here. Also: empty repo.
I'll remove this line.